api-docs/paths/scim/member_details.json

```diff
@@ -0,0 +1,211 @@
+{
+    "get": {
+        "tags": [
+            "SCIM"
+        ],
+        "description": "Query an individual organization member with a SCIM User GET Request.\n- The `name` object will contain fields `firstName` and `lastName` with the values of `N/A`. Sentry's SCIM API does not currently support these fields but returns them for compatibility purposes.",
+        "operationId": "Query an Individual Organization Member",
+        "parameters": [
+            {
+                "name": "organization_slug",
+                "in": "path",
+                "required": true,
+                "schema": {
+                    "type": "string"
+                }
+            },
+            {
+                "name": "member_id",
+                "in": "path",
+                "required": true,
+                "description": "The id of the member you'd like to query.",
+                "schema": {
+                    "type": "integer"
+                }
+            }
+        ],
+        "responses": {
+            "200": {
+                "description": "Success",
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "$ref": "../../components/schemas/scim/user.json#/User"
+                        },
+                        "example": [
+                            {
+                                "schemas": [
+                                    "urn:ietf:params:scim:schemas:core:2.0:User"
+                                ],
+                                "id": "102",
+                                "userName": "test.user@okta.local",
+                                "emails": [
+                                    {
+                                        "primary": true,
+                                        "value": "test.user@okta.local",
+                                        "type": "work"
+                                    }
+                                ],
+                                "name": {
+                                    "familyName": "N/A",
+                                    "givenName": "N/A"
+                                },
+                                "active": true,
+                                "meta": {
+                                    "resourceType": "User"
+                                }
+                            }
+                        ]
+                    }
+                }
+            },
+            "401": {
+                "description": "Unauthorized"
+            },
+            "403": {
+                "description": "Forbidden"
+            }
+        },
+        "security": [
+            {
+                "auth_token": [
+                    "member:read"
+                ]
+            }
+        ]
+    },
+    "patch": {
```

Comment:
the patch method ends up basically being the same as a DELETE -- okta doesn't support /User DELETE, only setting active to false. (which does the same thing as a delete for us since we don't support inactive org members currently, i have confirmed with their support that is okay, should be good for azure as well).